easybuild/easyconfigs/e/ESMF/ESMF-7.0.2-intel-2018a.eb

```diff
@@ -0,0 +1,32 @@
+name = 'ESMF'
+version = '7.0.2'
+
+homepage = 'http://sourceforge.net/projects/esmf'
+description = """The Earth System Modeling Framework (ESMF) is software for building and coupling weather,
+ climate, and related models."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'usempi': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%%(namelower)s_%s_src.tar.gz' % '_'.join(version.split('.'))]
+patches = ['ESMF-6.1.1_libopts.patch', 'intel-2018-qopenmp.patch']
```

Comment:
style nitpicking:

* one patch per line please
* rename new patch file to reflect what software (version) the patch is for
* favor `-fopenmp` over `-qopenmp` since `-fopenmp` works with both GCC & Intel compilers (so it will be supported in the foreseeable future in Intel compilers, I'm sure)

```python
patches = [
    'ESMF-6.1.1_libopts.patch',
    'ESMF-%(version)s-intel-fopenmp.patch',
]